go/teams/storage/supports_hidden_flag.go

```diff
@@ -0,0 +1,90 @@
+package storage
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/keybase1"
+)
+
+// SupportsHiddenFlagStorage stores the supports_hidden_chain flag to disk and memory.
+type SupportsHiddenFlagStorage struct {
+	*storageGeneric
+}
+
+// Increment to invalidate the disk cache.
+const supportsHiddenFlagDiskStorageVersion = 1
+const supportsHiddenFlagCacheLRUSize = 200
```

Comment:
Could make it bigger. For bots

Reply:
What do you think is a good value here? 1k? 10k?